presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/env/DockerContainer.java

```diff
@@ -23,10 +23,12 @@
 import com.google.common.base.Stopwatch;
```

Comment:
> Make more operations async

What is the benefit of this?

Reply:
Previously some of the operations were not interruptible + they were executed in thread pool not controlled by launcher (okhttp from testcontainers)